hack/build/docker/cdi-apiserver/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM fedora:27
```

Comment:
I am seeing 2 fedora:27s and one fedora28 in these docker files, we are going to have to make these consistent at some point.

Reply:
Good catch!  Will bump those two to 28.